src/main/java/roomescape/reservation/ReservationController.java

```diff
@@ -0,0 +1,36 @@
+package roomescape.reservation;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import roomescape.entities.Reservation;
+import roomescape.reservation.data.ReservationAddRequestDTO;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+    private final ReservationService reservationService;
+
+    @Autowired
+    public ReservationController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
```

Comment:
`@Autowired` 어노테이션을 사용해주셨군요!
이 어노테이션은 필드, 생성자 등 붙일 수 있는 곳이 여러 개 있죠.
그 중 생성자는 꼭 붙이지 않아도 빈 주입이 되는데요, 필수로 붙일 필요가 없다면 제거해보는건 어떨까요?

```suggestion
    public ReservationController(ReservationService reservationService) {
        this.reservationService = reservationService;
    }
```